cmd/broker/filter/main.go

```diff
@@ -0,0 +1,85 @@
+/*
+ * Copyright 2019 The Knative Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package main
```

Comment:
General comment on broker-filter. IMO, it isn't intuitive to understand its significance from its name. My first impression was that this is the ingress filter. It took me some time going over how K8 services are defined to understand that his is the guy that handles individual triggers. 
Could we rename it to something Trigger-specific like TriggerHandler?
If you concur then the comment applies to other related objects too such as roles, env variables etc.

Thoughts?

Reply:
We should probably rename all of these pieces to whatever Trigger ends up named as. As the final naming decision hasn't been made, I will leave it as-is and we can change it once later.